mybatis-spring-boot-test-autoconfigure/pom.xml

```diff
@@ -30,18 +30,37 @@
   <name>mybatis-spring-boot-test-autoconfigure</name>
   <dependencies>
     <dependency>
-      <groupId>org.mybatis.spring.boot</groupId>
-      <artifactId>mybatis-spring-boot-starter</artifactId>
+      <groupId>org.springframework.boot</groupId>
```

Comment:
not sure if I understood very well. But theses dependencies `spring-boot-test-autoconfigure` and `spring-test` in a real project will be provided by `spring-boot-starter-test`, isn't it?